.circleci/config.yml

```diff
@@ -386,6 +387,22 @@ workflows:
           filters:
             branches:
               only: /^dependabot/submodules/.*/
+      - ios-device-checks:
+          name: Test iOS on Device - Full (On Develop)
+          post-to-slack: false
+          filters:
+            branches:
+              only: 
+                - develop
+                - try/running-ui-tests-on-updated-develop
+      - android-device-checks:
+          name: Test Android on Device - Full (On Develop)
+          post-to-slack: false
```

Comment:
Is `post-to-slack` false just for testing, or is the plan to leave it as false when we merge this?

Reply:
The plan was to set it to `true` before merging, but I forgot to mention that in the PR description. I'll update that now, thanks for noticing!